### PR TITLE
Migrating Vault endpoint to new directories as per magedoc3978

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) moves the Valut endpoint to the new queries and mutations subdir.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/vault.html

### CODE REVIEW

File: guides/v2.3/graphql/mutations/delete-payment-token.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+`mutation: {deletePaymentToken(public_hash) {DeletePaymentTokenOutput}}`
+
+## Example usage
+
+The following example deletes the Discover Card listed in the results of the `customerPaymentTokens` query.
+
+**Request**
+
+``` text
```
Comment (keharper) [inline guides/v2.3/graphql/mutations/delete-payment-token.md]:
```suggestion
```graphql
```

File: guides/v2.3/graphql/queries/customer-payment-tokens.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+`{customerPaymentTokens{CustomerPaymentTokens}}`
+
+## Example usage
+
+The following example returns all the current customer's payment tokens.
+
+**Request**
+
+```text
```
Comment (keharper) [inline guides/v2.3/graphql/queries/customer-payment-tokens.md]:
```suggestion
```graphql
```

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
running tests

**Pull request merged**